### ISSUE TITLE
does the recent 15*192 nets catch up with latest 12*256 nets

### ISSUE THREAD

**Issue by l1t1** [CuriosAI/sai#121]
there was not a test match of them for many days

**Comment by Vandertic** [comment]
You are right. The standard training procedure did not give a very good result. We are training in a different way and for now there's no promising network. They seem quite weak.

**Comment by l1t1** [comment]
any news?

**Comment by nutpen85** [comment]
Congratulations, I just saw the first 15-block-network beat LZ-117. Awesome job.
